frontend/components/Auction.vue

```diff
@@ -331,6 +358,14 @@ export default Vue.extend({
                 }
             },
         },
+        fetchTakeEvents: {
+            immediate: true,
+            handler() {
+                if (!this.areAuctionsFetching && !this.auction) {
+                    this.$emit('fetchTakeEventsFromAuction', this.auctionId);
+                }
+            },
+        },
```

Comment:
This will fetch events no matter if auction is found or not, on the component load and never again, since there is no prop or data called `fetchTakeEvents`. Instead, you should watch for the specific prop:
```suggestion
        areAuctionsFetching(areAuctionsFetching) {
            if (!areAuctionsFetching && !this.auction) {
                this.$emit('fetchTakeEventsFromAuction', this.auctionId);
            }
        },
```